plugin/evm/vm.go

```diff
@@ -701,6 +701,7 @@ func (vm *VM) initBlockBuilding() error {
 		ethTxPushGossiper, err = gossip.NewPushGossiper[*GossipEthTx](
 			ethTxGossipMarshaller,
 			ethTxPool,
+			vm.validators,
 			ethTxGossipClient,
 			ethTxGossipMetrics,
 			pushGossipParams,
```

Comment:
Should we be adding the Stake percentage into the BranchingFactor here?

Reply:
done